asmcli/lib/checks.sh

```diff
@@ -464,12 +464,14 @@ istio_namespace_exists() {
 }
 
 is_autopilot() {
-  # TODO: wait till https://pkg.go.dev/google.golang.org/genproto/googleapis/container/v1#Cluster  
-  # publish the `Autopilot` field
-  # This is a temporary workaround to check if a cluster is Autopilot or GKE
-  # This CRD will be installed only if the cluster is Autopilot
   if [[ "${IS_AUTOPILOT}" -eq 1 ]]; then return; fi
-  if ! retry 2 kubectl get crd allowlistedworkloads.auto.gke.io 1>/dev/null 2>/dev/null; then
+
+  local _IS_AUTOPILOT; _IS_AUTOPILOT="$(gcloud container clusters describe "${CLUSTER_NAME}" \
```

Comment:
this is what worked for me:
```
gcloud container clusters  describe autopilot-cluster-1 --region ${LOCATION} --project ${PROJECT_ID} --format json | jq .autopilot.enabled
```
do we need to specify the format to be `json`?

Reply:
yes, added